pylintrc

```diff
@@ -12,6 +12,7 @@
 disable=C0301,  #line too long
         C0103,  #constants uppercase
         import-error, # unless youre in the venv, it won't find many packages
+        F722  # type generation induces these
```

Comment:
You might be able to use a [root template](https://koxudaxi.github.io/datamodel-code-generator/custom_template/) to add pylint annotations to the generated file. That way you wouldn't need to ignore this lint repo-wide

Reply:
ill add that as an issue, danke